Messaging.md

```diff
@@ -87,33 +97,123 @@ The following DLC-specific types are used throughout the specification. All type
 
 ### The `contract_info` Type
 
-This type contains information about a contracts outcomes and their corresponding payouts. To save space, only one side's POV is included in this message as the other can be derived using `remote_payout = total_collateral - local_payout`.
+This type contains information about a contract's outcomes, their corresponding payouts, and the oracles to be used.
 
 #### Version 0 `contract_info`
 
-1. type: 42768 (`contract_info_v0`)
+1. type: 55342 (`contract_info_v0`)
 2. data:
-   * [`bigsize`:`nb_outcomes`]
-   * [`sha256`:`outcome_1`]
-   * [`u64`:`outcome_1_local_payout`]
+   * [`u64`:`total_collateral`]
```

Comment:
IIRC before you were against putting this to allow the accepting party to set the collateral they want to put, do we give up on that (I'm fine either way, just to understand).

Reply:
yup, this is me with a white flag, at least for now